media-sound/flacon/flacon-2.1.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+# Ignore rudimentary et, uz@Latn, zh_TW translation(s).
+PLOCALES="cs cs_CZ de es es_MX fr gl hu it ja_JP lt nb nl pl pl_PL pt_BR pt_PT ro_RO ru sr sr@latin tr uk zh_CN"
+
+inherit cmake-utils eutils gnome2-utils l10n virtualx xdg-utils
+
+DESCRIPTION="Extracts audio tracks from an audio CD image to separate tracks"
+HOMEPAGE="https://flacon.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qt5 test"
+
+COMMON_DEPEND="
+	dev-libs/uchardet
+	!qt5? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
```

Comment:
while you're at it, could you also drop Qt4 support? The KDE team will drop Qt4 soon, and the sooner packages stop depending on it, the better. Qt4 is not a feature anymore, it's a liability. See also https://github.com/gentoo/gentoo/pull/2605#issuecomment-255518670